MaskChannels.py

```diff
@@ -6,15 +6,14 @@
 @author: Geng
 
 """
-
-from ROOT import gROOT, gDirectory, TMultiGraph, TNamed, TLegend, TCanvas, TGraph, TH1F, TH2F, TFile, TDirectory #Classes
-from ROOT import kGreen, kYellow, kBlue, kRed #Colors
+from ROOT import *
```

Comment:
Don't need both line 9 and 10.
Probably it's simplest to remove line 10, and change line 11 to be:
```gROOT.SetBatch(True)```